plugins/clnrest/utilities/rpc_routes.py

```diff
@@ -0,0 +1,71 @@
+import json5
+from flask import request, make_response, Response, stream_with_context
+from flask_restx import Namespace, Resource
+from .shared import call_rpc_method, verify_rune, process_help_response
+from .rpc_plugin import plugin
+
+methods_list = []
+rpcns = Namespace("RPCs")
+payload_model = rpcns.model("Payload", {}, None, False)
+
+@rpcns.route("/list-methods")
+class ListMethodsResource(Resource):
+    @rpcns.response(200, "Success")
+    @rpcns.response(500, "Server error")
+    def get(self):
+        """Get the list of all valid rpc methods"""
+        try:
+            help_response = call_rpc_method(plugin, "help", [])
+            html_content = process_help_response(help_response)
+            response = make_response(html_content)
+            response.headers["Content-Type"] = "text/html"
+            return response
+
```

Comment:
```suggestion
```

Reply:
Thank you. Checked and removed extra lines from all files.